pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -23,10 +23,13 @@ import (
 	"github.com/kinvolk/lokomotive/internal/template"
 	"github.com/kinvolk/lokomotive/pkg/components"
 	"github.com/kinvolk/lokomotive/pkg/components/util"
+	"github.com/kinvolk/lokomotive/pkg/k8sutil"
 )
 
-const name = "aws-ebs-csi-driver"
-
+const (
+	name             = "aws-ebs-csi-driver"
+	releaseNamespace = "kube-system"
```

Comment:
I wouldn't put it in the const, I think that reduces the visibility. As we should treat `Metadata()` function as const anyway, it's fine to put the strings there. I'd even consider putting `name` const there at some point in the future :)